new-integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/LoggingUtil.java

```diff
@@ -137,4 +232,162 @@ private static void writeToFile(Object obj, String resultDir, String fileName) t
     }
   }
 
-}
+  /**
+   * Copy files from persistent volume to local folder.
+   * @param namespace name of the namespace
+   * @param hostPath the persistent volume host path
+   * @param destinationPath destination folder to copy the files to
+   * @throws ApiException when pod interaction fails
+   */
+  private static void copyFromPV(String namespace, String hostPath, Path destinationPath) throws ApiException {
+    V1Pod pvPod = null;
+    try {
+      // create a temporary pod to get access to the interested persistent volume
+      pvPod = setupPVPod(namespace, hostPath);
+      copyDirectoryFromPod(pvPod, hostPath, destinationPath);
+    } finally {
+      // remove the temporary pod
+      if (pvPod != null) {
+        cleanupPVPod(namespace);
+      }
+    }
+  }
+
+  /**
+   * Creates temporary pod with persistent volume claim and persistent volume using host path.
+   *
+   * @param namespace name of the namespace
```

Comment:
what namespace?

Reply:
modified the description to give more details